### PR TITLE
Add apierrors as alias for k8s.io/apimachinery/pkg/api/errors

### PR DESCRIPTION
**Pull request by k-toyoda-pi**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Many files under `test/` import `"k8s.io/apimachinery/pkg/api/errors"`.
But, different aliases are defined for each file.

For example,
```
test/e2e/auth/service_accounts.go:
        apierrors "k8s.io/apimachinery/pkg/api/errors"

test/e2e/auth/pod_security_policy.go:
        apierrs "k8s.io/apimachinery/pkg/api/errors"

test/e2e/auth/audit_dynamic.go:
        "k8s.io/apimachinery/pkg/api/errors"
```
```
$ grep -rs \"k8s.io/apimachinery/pkg/api/errors\" test/ | grep apierrors | wc -l
15
$ grep -rs \"k8s.io/apimachinery/pkg/api/errors\" test/ | grep apierrs | wc -l
32
$ grep -rs \"k8s.io/apimachinery/pkg/api/errors\" test/ | egrep -v "apierrors|apierrs" |  wc -l
60
```
We can get more readability if the aliases are unified.
Therefore, this PR adds `apierrs` to `hack/.import-aliases` being used with `hack/verify-import-aliases.sh`.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Comment by k-toyoda-pi** [comment]
https://github.com/kubernetes/kubernetes/pull/85009 enabled hack/verify-import-aliases.sh on CI.
So, we should resolve the aliases before this PR will be marged.
/hold

**Comment by danielqsj** [comment]
@k-toyoda-pi @fejta 

Retrieved the entire kubernetes project, the most common alias usage is `apierrors`:
```
grep -rs \"k8s.io/apimachinery/pkg/api/errors\" ./ | grep apierrors | wc -l
     200

grep -rs \"k8s.io/apimachinery/pkg/api/errors\" ./ | grep apierrs | wc -l
      51
```
So I suggest to use `apierrors` instead of `apierrs`.

**Comment by k-toyoda-pi** [comment]
@danielqsj Thank you for the suggestion and modification in #85123. I will replace `apierrs` with `apierrors` on this PR after #85123 is merged.

File: hack/.import-aliases
Comment (fejta) [inline hack/.import-aliases]: apierrors

**Review by danielqsj — APPROVED** [review]
/lgtm

**Comment by k-toyoda-pi** [comment]
@danielqsj Thank you for lgtm. I'm waiting for ci jobs being completed.

**Comment by k-toyoda-pi** [comment]
pull-kubernetes-verify failed because endpointslice.go doesn't have alias for api/errors. So, I created PR #86655 for adding it. I make this PR hold until #86655 is merged.
/hold

**Pull request merged**
